_config.yml

```diff
@@ -52,6 +55,20 @@ defaults:
       docs_base: "/docs/fluo/1.2"
       javadoc_base: "https://static.javadoc.io/org.apache.fluo/fluo-api/1.1.0-incubating"
       github_base: "https://github.com/apache/fluo/blob/master"
+  -
+    scope:
+      path: "_recipes-1-2"
+      type: "recipes-1-2"
+    values:
+      layout: "recipes-1.2"
+      title_prefix: "Fluo Recipes Documentation - "
+      version: "1.2.0"
+      minor_release: "1.2"
+      docs_base: "/docs/fluo-recipes/1.2"
+      github_base: "https://github.com/apache/fluo-recipes/blob/master"
+      javadoc_fluo: "https://static.javadoc.io/org.apache.fluo/fluo-api/1.1.0-incubating"
+      javadoc_core: "https://static.javadoc.io/org.apache.fluo/fluo-recipes-core/1.1.0-incubating"
```

Comment:
This name is a bit ambiguous, it could be recipes core or fluo core.  I am ok w/ not changing it.  When seeing this in other places I found myself wondering which core it was for.

Reply:
I am OK with changing it but I don't know a better name without being long.